include/alpaka/atomic/AtomicUniformCudaHip.hpp

```diff
@@ -71,7 +71,7 @@ inline namespace alpakaGlobal
             decltype(atomicCAS(alpaka::core::declval<T*>(), alpaka::core::declval<T>(), alpaka::core::declval<T>()))>>
         : std::true_type
     {
-        __device__ static T atomic(T* add, T compare, T value)
+        static __device__ T atomic(T* add, T compare, T value)
```

Comment:
I somehow prefer to have the execution space specifiers on the LHS. Can we achieve this with clang-format?

Reply:
Unfortunately not (or I didn't find the right option). I played around a bit and clang-format will always place `static` in front of `__device__`.